presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueryFramework.java

```diff
@@ -294,6 +294,11 @@ protected static void skipTestUnless(boolean requirement)
         }
     }
 
+    protected QueryRunner getQueryRunner()
+    {
+        return requireNonNull(queryRunner, "queryRunner is null");
```

Comment:
Make this `checkState`. This is a better exception since it means the method was invoked at the wrong time.